pyproject.toml

```diff
@@ -119,6 +119,8 @@ Werkzeug = "3.0.1"
 toml = "^0.10.2"
 webargs = "^8.2.0"
 WTForms = "^3.0.1"
+attr = "^0.3.2"
```

Comment:
Why adding those two dependencies? I don't see them imported anywhere.

Reply:
From the `Tox-docs CI`, There was an issue with `filters.py` inside `/workspace/src/github.com/fedora-infra/anitya/.tox/docs/lib/python3.11/site-packages/attr/.` This problem is occurring during an attempt to install idna, and attrs using Poetry. 
To fix that issue I installed manually both dependencies.

<img width="1582" alt="Screenshot 2024-03-13 at 14 20 44" src="https://github.com/fedora-infra/anitya/assets/82499435/d834a854-25f8-4649-b4b4-15d280d13c67">

Reply:
This isn't related to Anitya project but it's issue in dependencies of pip or poetry. Those shouldn't be in the dependencies for Anitya as we don't use them. We should only update poetry.lock file.

Reply:
Let me do that in separate PR.

Reply:
The PR is now available https://github.com/fedora-infra/anitya/pull/1740. Could you remove changes in `poetry.lock` and `pyproject.toml` from this PR.